frontend/packages/ceph-storage-plugin/src/components/block-pool/body.tsx

```diff
@@ -0,0 +1,258 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import * as _ from 'lodash';
+
+import {
+  Alert,
+  Dropdown,
+  DropdownToggle,
+  DropdownItem,
+  EmptyState,
+  EmptyStateIcon,
+  EmptyStateBody,
+} from '@patternfly/react-core';
+import { CaretDownIcon } from '@patternfly/react-icons';
+import { ListKind } from '@console/internal/module/k8s';
+import { useK8sGet } from '@console/internal/components/utils/k8s-get-hook';
+import { useFlag } from '@console/shared/src/hooks/flag';
+
+import { CephClusterKind, StorageClusterKind } from '../../types';
+import { OCSServiceModel } from '../../models';
+import { CEPH_STORAGE_NAMESPACE, OCS_DEVICE_REPLICA } from '../../constants/index';
+import { checkArbiterCluster } from '../../utils/common';
+import { PROGRESS_STATUS } from '../../utils/storage-pool';
+import { POOL_STATE, POOL_PROGRESS } from '../../constants/storage-pool-const';
+import {
+  BlockPoolState,
+  BlockPoolAction,
+  BlockPoolActionType,
+  getErrorMessage,
+} from '../../utils/block-pool';
+import { GUARDED_FEATURES } from '../../features';
+
+import './body.scss';
+
+export const PoolStatus: React.FC<PoolStatusProps> = ({ status, name, error = '' }) => {
```

Comment:
```suggestion
export const PoolStatus: React.FC<PoolStatusProps> = ({ status, name, error = '' }) => {
```
There is some inconsistency in the naming conventions:
Somwhere we are saying `Pool*` and somehere `StoragePool*`.
Lets keep all components as `BlockPool*` - i.e `BlockPoolBody, BlockPoolFooter, BlockPoolModal, etc.

Reply:
ack